source/stylesheets/modules/_technical-documentation.scss

```diff
@@ -3,9 +3,9 @@
   max-width: 40em;
 
   line-height: 1.6;
-  
+
   @include media(tablet) {
-    margin: 0 $gutter $gutter;
+    margin: 0 ($gutter + 2px) $gutter;
```

Comment:
This could do with a comment or a variable to explain what that extra 2px is for.